### DISCUSSION TITLE
请问排盘可视化方面的代码开源么？

### DISCUSSION THREAD

**Discussion by MillerTheBest** [SylarLong/iztro#175] [Q&A]
请问排盘可视化方面的代码开源么？也就是前端网页上，提交生日之后submit，生成了palaces，使用palaces生成12宫可视化显示12宫以及每个宫内的内容这一步的代码。谢谢。

**Comment by SylarLong** [comment]
@MillerTheBest  已经开源了，请参考 https://github.com/SylarLong/react-iztro

**Reply by MillerTheBest** [reply]
谢谢！
